server/NucleiSegmentation/NucleiSegmentation.py

```diff
@@ -17,15 +18,33 @@
 }
 
 
+def read_image(file):
+    return skimage.io.imread(file)[:, :, :3]
+
+
 def main(args):
 
     #
     # Read Input Image
     #
     print('>> Reading input image')
 
-    imInput = skimage.io.imread(args.inputImageFile)[:, :, :3]
-
+    imInput = None
+    imageFile = args.inputImageFile
+    if os.path.isdir(imageFile):
+        dir = imageFile
+        for file in os.listdir(dir):
```

Comment:
@zach @jbeezley What is the best/cleanest way to fix this?

In the REST end-point of a CLI, should i request the girder ID of an item or a file for an input argument of type image?